spec/controllers/organizations_controller_spec.rb

```diff
@@ -34,6 +35,34 @@
       end
     end
 
+    context 'user with admin privilege on the organization but not part of the classroom' do
+      before(:each) do
+        organization.users = []
```

Comment:
since the factory creates an org with a user, I believe this needs a call to `organization.save!`